docs/outputs.rst

```diff
@@ -22,207 +21,330 @@ FMRIPrep generates three broad classes of outcomes:
+can be added as a regressor in group level analysis [Yan2013]_.
+
+*Spike regressors* for outlier censoring can also be generated from within *fMRIPrep* using
+the command line options ``--fd-spike-threshold`` and ``--dvars-spike-threshold``
+(default: FD > 0.5 mm or DVARS > 1.5). Spike regressors are stored in separate ``motion_outlier_XX``
+columns.
+
+ICA-AROMA confounds
+===================
+
+- ``aroma_motion_XX`` - the motion-related components identified by :abbr:`ICA (independent components analysis)`
+  -:abbr:`AROMA (Automatic Removal Of Motion Artifacts)` (if enabled with a flag ``--use-aroma``) .
+
+.. warning::
+    If you are already using ICA-AROMA cleaned data (``~desc-smoothAROMAnonaggr_bold.nii.gz``),
+    do not include ICA-AROMA confounds during your design specification or denoising procedure.
+
+CompCor confounds
+=================
+:abbr:`CompCor (Component Based Noise Correction)` is a component-based noise correlation method.
+In the method, principal components are calculated within an :abbr:`ROI (Region of Interest)`
+that is unlikely to include signal related to neuronal activity, such as :abbr:`CSF (cerebro-spinal fluid)`
+and :abbr:`WM (white matter)` masks.
+Signals extracted from CompCor components can be further regressed out from the fMRI data with a
+denoising procedure [Behzadi2007]_.
+
+- ``a_comp_cor_XX`` - additional noise components are calculated using anatomical :abbr:`CompCor
+  (Component Based Noise Correction)`;
+- ``t_comp_cor_XX`` - additional noise components are calculated using temporal :abbr:`CompCor
+  (Component Based Noise Correction)`.
 
 Four separate CompCor decompositions are performed to compute noise components: one temporal
-decomposition and three anatomical decompositions across three different noise ROIs: an eroded
-white matter compartment, an eroded CSF compartment, and a combined mask derived from the union
-of these.
-In general, only a subset of these decompositions should be used for further denoising.
-The original Behzadi aCompCor implementation ([Behzadi2007]_) can be applied using
-components from the combined ROI, while the more recent Muschelli implementation
-([Muschelli2014]_) can be applied using the WM and CSF ROIs.
-To determine the provenance of each component, consult the metadata file (see below).
-
-All these confounds can be used to perform *scrubbing* and *censoring* of outliers,
-in the subsequent first-level analysis when building the design matrix,
-and in group level analysis.
-*Spike regressors* for outlier censoring can also be generated from within fMRIPrep using
-the command line options ``--fd-spike-threshold`` and ``--dvars-spike-threshold``.
-Spike regressors are stored in separate ``motion_outlier_XX`` columns.
+decomposition (``t_comp_cor_XX``) and three anatomical decompositions (``a_comp_cor_XX``) across
+three different noise ROIs: an eroded white matter mask, an eroded CSF mask, and a combined mask derived
+from the union of these.
```

Comment:
We need to mention that CompCor is performed on the time series with non-steady-state volumes discarded and detrended with a high-pass filter (cosine basis, 128s cutoff). Therefore, in order to use these confounds, non-steady-state and cosine regressors MUST be included.